plugin/pkg/admission/storage/volumeattributesclass/setdefault/admission.go

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
If we have retroactive default class feature, do we actually need an admission controller? @jsafrane wdyt?

Reply:
Discussed in today's csi standup. We agreed the admission controller is not necessary if the PV controller is going to be applying the defaults.

Reply:
@msau42 Does this mean the set-default admission plugin of the storage class will also be deprecated and removed in the future?

Reply:
It's a possibility. I think we could use VAC to validate that PV controller changes are sufficient enough and there aren't corner cases we're missing by not having an admission controller.